Xamarin.PropertyEditing.Mac/Controls/EntryPropertyEditor.cs

```diff
@@ -2,9 +2,34 @@
 using AppKit;
 using Foundation;
 using Xamarin.PropertyEditing.ViewModels;
+using ObjCRuntime;
 
 namespace Xamarin.PropertyEditing.Mac
 {
+	class TextNextResponderDelegate : NSTextFieldDelegate
```

Comment:
Can we change the prefix? `TextNextResponderDelegate` doesn't make sure sense to me. `DelegatedRowTextFieldDelegate` or something based on the ProxyResponder would be nice.

Reply:
fixed